src/command_modules/azure-cli-interactive/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+0.3.16
+++++++
+* Point to azureProfile.json in .azure directory.
```

Comment:
I would recommend: "Fix issue where user is prompted to login when using interactive mode in Cloud Shell."